backend/app/api/endpoints/knowledge_code_wiki.py

```
@@ -408,6 +441,92 @@ def _assert_caller_may_regenerate(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="Only the Knowledge Base owner or a system administrator may configure automatic updates",
+        )
+
+
+@router.get(
+    "/{knowledge_base_id}/code-wiki/automatic-update",
+    response_model=CodeWikiAutomaticUpdate,
+)
+def get_code_wiki_automatic_update(
+    knowledge_base_id: int,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.16.3)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)